### ISSUE TITLE
Copying a config file from app A to app B on the same server? 

### ISSUE THREAD

**Issue by shartwell1** [Bre77/badrcm#7]
Q.  Is it possible to config copy an entire config file (e.g. savedsearches.conf) from one app to another on the same server?
Use case is get around Splunk Cloud uploads to the "default" folder (which can't be deleted) by doing something like this: 

appA/default/savedsearches.conf  --> appB/local/savedsearches.conf

Thanks in advance!

**Comment by Bre77** [comment]
Yes that definitely should be possible using the copy page and selecting the same server in both columns. The app will always save the changes to local.

**Comment by shartwell1** [comment]
Good news…but when I try this exact thing, it won’t change that first line below which looks like the app A version in app.conf.  It says App A 1.0.0 and second column says the same.  Maybe a bug?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Bre77** [comment]
@shartwell1 Ah yes, unfortuantely you are correct, the copy page does maintain the app context when copying which does not meet your needs. The terrible way to achieve this would be to connect a Splunk Enterprise instance to the app, migrate the configuration there, rename the app using SSH or Config Explorer, and then migrate it back to the Cloud. You could also migrate it back to the same app after deleting it in cloud to get it out of default.

**Comment by shartwell1** [comment]
Thanks - ok cool - so shortest path would be create searches in local context in Enterprise installation and copy to empty app of same name in Cloud? 🤔

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
